src/app/modules/welcome-screen/welcome-screen.component.css

```diff
@@ -0,0 +1,15 @@
+.big-bold {
```

Comment:
Label name should be informing about functionality of html component, not about css functionality. In this case sth like "title-label". In more advanced htmls, when there is multiple titles css label name could look like "title__content-label", "title__wrapper-label" etc